help/rename-a-channel.md

```diff
@@ -39,3 +39,9 @@ and Unicode emoji.
 {end_tabs}
 
 {!automated-notice-channel-event.md!}
+
+## Related articles
+
+* [User roles](/help/user-roles)
+* [Channel permissions](/help/channel-permissions)
+* [Change a channel's description](/help/change-the-channel-description)
```

Comment:
I would just do:

* [Change a channel's description](/help/change-the-channel-description)
* [Channel permissions](/help/channel-permissions)

Reply:
(dropping roles)